engines/aws.rb

```diff
@@ -0,0 +1,91 @@
+require 'aws-sdk-translate'
+require 'aws-sdk-comprehend'
+require_relative 'translator'
```

Comment:
Is it right to make it require_relative?.

Reply:
Yes. That way I would like to avoid conflict with potentially similar-ly named gems